src/Robo/Plugin/Commands/DevelopmentModeBaseCommands.php

```diff
@@ -347,11 +347,12 @@ protected function frontendDevEnableDrupal(string $siteDir = 'default', array $o
             );
         }
 
-        $this->say("enablig twig.debug in development.services.yml.");
+        $this->say("enablig twig.debug in development services yml config.");
```

Comment:
```suggestion
        $this->say("optimizing twig for front-end development in development services yml config.");
```

My thinking here is that we are changing three settings, not just debug. In your previous version, I _liked_ that the file was specified (though I suppose it should have been `fe.development.services.yml`) as opposed to making a general statement about the file. So... opinions.